src/main/resources/view/DialogBox.fxml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.image.ImageView?>
+<?import javafx.scene.layout.HBox?>
+
+<fx:root alignment="TOP_RIGHT" maxHeight="1.7976931348623157E308" maxWidth="1.7976931348623157E308" prefWidth="400.0" stylesheets="@../css/dialog-box.css" type="javafx.scene.layout.HBox" xmlns="http://javafx.com/javafx/22" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
Small nitpick but I don't think your .fxml is working? Your .fxml is stating it using java SDK vers 22, usually because every time you use scenebuilder to save the edits onto your .fxml file, it will set the version as xmlns="http://javafx.com/javafx/22"  instead of xmlns="http://javafx.com/javafx/17", so need to edit this file every time you use scenebuilder

Reply:
Oh okay I did not even know this. Thank you!